MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -202,6 +212,8 @@ - (void)flush {
 }
 
 - (void)sendTurnstileEvent {
+    [self.configurationUpdater updateConfigurationFromAPIClient:self.apiClient];
```

Comment:
Does the rest of `sendTurnstileEvent` (or any event for that matter) need to happen once the configuration is complete?

Reply:
Negative. There aren't any plans currently for configurations to affect events directly.

Reply:
👍